### PR TITLE
Types for Accordion Group

### PR DESCRIPTION
**Pull request by GomezIvann**

### CODE REVIEW

File: projects/dxc-ngx-cdk/src/lib/dxc-accordionGroup/dxc-accordionGroup.component.ts [hunk 1]
```diff
@@ -23,27 +23,49 @@ import {
...
-    } else if (value === null || value === "null") {
-      this._indexActive = null;
-    } else {
-      this._indexActive = coerceNumberProperty(value);
-    }
+  set indexActive(value: number) {
+    if (value == null) this._indexActive = value;
+    else this._indexActive = coerceNumberProperty(value);
   }
   private _indexActive;
```
Comment (Lisa159) [inline projects/dxc-ngx-cdk/src/lib/dxc-accordionGroup/dxc-accordionGroup.component.ts:65]: I would initialize indeaxActive.

Comment (GomezIvann) [reply projects/dxc-ngx-cdk/src/lib/dxc-accordionGroup/dxc-accordionGroup.component.ts:65]: Let me ask @aweell. In the Accordion Group component, should the first option appear expanded when it is uncontrolled (what accordion is expanded is managed internally by us)

Comment (Lisa159) [reply projects/dxc-ngx-cdk/src/lib/dxc-accordionGroup/dxc-accordionGroup.component.ts:65]: I think everything should be collapsed in the beginning.

Comment (mawelCaballero) [reply projects/dxc-ngx-cdk/src/lib/dxc-accordionGroup/dxc-accordionGroup.component.ts:65]: According to @aweell , everything is collapsed in the initialization.

**Pull request merged**